src/shared/components/FilterList/MinimalistInjectionOption.scss

```diff
@@ -0,0 +1,32 @@
+@import '@influxdata/clockface/dist/variables.scss';
+
+// inherits from the other flux-toolbar--list-item class
+// overwrites certain props
+.flux-toolbar--list-item {
```

Comment:
Choose to re-use the same classNames (from the shared components), and overwrite what props we need different here. Goal is to try keeping the look as much in sync as possible...except where we have explicitly updated design for this MinimalistInjectionOption.